### PR TITLE
WIP: Update README

### PR DESCRIPTION
**Pull request by markwolff**

### CODE REVIEW

File: AISKU/API.md [hunk 1]
```diff
@@ -0,0 +1,770 @@
...
+
+```ts
+config: IConfig
+```
+
+Values that control how the telemetry data is sent.
+
+#### Core config
+```ts
+interface IConfiguration {
```
Comment (jpiyali) [inline AISKU/API.md]: Two IConfiguration?

File: Examples/react-application-insights-demo/public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
+  "icons": [
```
Comment (jpiyali) [inline Examples/react-application-insights-demo/public/manifest.json]: what does the user need to install to get this to run? Do you need a package.json?

File: Examples/react-application-insights-demo/src/serviceWorker.js [hunk 3]
```diff
@@ -0,0 +1,135 @@
+// This optional code is used to register a service worker.
```
Comment (jpiyali) [inline Examples/react-application-insights-demo/src/serviceWorker.js]: what is this sample trying to demonstrate?

Comment (jpiyali) [reply Examples/react-application-insights-demo/src/serviceWorker.js]: Where is it using application insights?

File: AISKU/API.md (hunk 1 above)
Comment (markwolff) [reply AISKU/API.md]: Removed both. Leaving only in README

File: Examples/react-application-insights-demo/public/manifest.json (hunk 2 above)
Comment (markwolff) [reply Examples/react-application-insights-demo/public/manifest.json]: Added a note in README on how the whole repo is just generated code from create-react-app except for App.jsx

**Pull request merged**